presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/DetermineJoinDistributionType.java

```diff
@@ -138,5 +139,10 @@ private static boolean isCrossJoin(JoinNode node)
 
             return SemiJoinNode.DistributionType.REPLICATED;
         }
+
+        private static boolean isDistributedJoinEnabled(Session session)
+        {
+            return !getJoinDistributionType(session).equals(FeaturesConfig.JoinDistributionType.REPLICATED);
```

Comment:
@findepi that's what this does
@sopel39 I'll rename to isRepartitionedJoinEnabled.  It does get used, but you don't see the changes because it replaced the static import from the old SystemSessionProperties.isDistributedJoinEnabled.  

Also, (in a different commit) I'll have to make this class interact correctly with ReorderJoins.  we don't want it to overwrite specifically chosen join distribution types.